### DISCUSSION TITLE
Development stopped, no new modifications？

### DISCUSSION THREAD

**Discussion by zhang-wenchao** [rwf2/Rocket#1957] [Questions]
I found that I haven't changed for 2 months, can I continue to use this project? Is there a subsequent maintenance release plan?

**Comment by SergioBenitez** [comment]
I have not abandoned the project. I hope to share more soon.

**Reply by SpootDev** [reply]
Whew

**Reply by beemstream** [reply]
Thanks for all the hard work regarding this, I love Rocket!

**Reply by dharmatech** [reply]
@SergioBenitez,

Loving rocket. I'm enjoying exploring this wonderful framework you've poured so much into.

Take all the time you need man. No rush. We'll all be here to continue the Rocket launch journey whenever you're ready.

**Reply by h8f1z** [reply]
@SergioBenitez I am seeing only 1 open 'issue' in the milestone to release 0.5: _pastebin example in documentation is outdated #1756._
You have already provided an updated pastebin link there. 
I am wondering why you haven't closed it and released the 0.5 of rocket.
I love rocket but there are few differences in 0.5 and 0.5rc.
I am eagerly waiting for 0.5 release.

Also, thank you for the amazing work.
